Examples/ObjectiveC/LiveDataExample.m

```diff
@@ -0,0 +1,53 @@
+
+#import "LiveDataExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLiveData = @"LiveDataExample";
+
+@interface LiveDataExample () <MGLMapViewDelegate>
+
+@property (nonatomic, strong, nullable) NSTimer *timer;
+@property MGLShapeSource *source;
+@end
+
+@implementation LiveDataExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Create a new map view using the Mapbox Dark style.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.tintColor = [UIColor darkGrayColor];
+    
+    // Set the map view‘s delegate property.
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    // Add a source to the map. https://wanderdrone.appspot.com/ generates coordinates for simulated paths.
+    NSURL *url = [NSURL URLWithString:@"https://wanderdrone.appspot.com/"];
+    _source = [[MGLShapeSource alloc] initWithIdentifier:@"wanderdrone-source" URL:url options:nil];
```

Comment:
There’s no need for `-source` in a source identifier and `-layer` in a layer identifier. Just thinking that developers might start [cargo-culting](http://www.catb.org/~esr/jargon/html/C/cargo-cult-programming.html) identifiers if they see this example.